experimental/css-has-pseudo/INSTALL-POSTCSS.md

```diff
@@ -0,0 +1,163 @@
+# Installing PostCSS
+
+[CSS Has Pseudo] runs in all Node environments, with special instructions for:
+
+| [Node](#node) | [PostCSS CLI](#postcss-cli) | [Webpack](#webpack) | [Create React App](#create-react-app) | [Gulp](#gulp) | [Grunt](#grunt) |
+| --- | --- | --- | --- | --- | --- |
+
+## Node
+
+Add [CSS Has Pseudo] to your project:
+
+```bash
+npm install css-has-pseudo --save-dev
+```
+
+Use it as a [PostCSS] plugin:
+
+```js
+const postcss = require('postcss');
+const postcssHasPseudo = require('css-has-pseudo');
+
+postcss([
+  postcssHasPseudo(/* pluginOptions */)
+]).process(YOUR_CSS /*, processOptions */);
+```
+
+## PostCSS CLI
+
+Add [PostCSS CLI] to your project:
+
+```bash
+npm install postcss-cli --save-dev
+```
+
+Use [CSS Has Pseudo] in your `postcss.config.js` configuration file:
+
+```js
+const postcssHasPseudo = require('css-has-pseudo');
+
+module.exports = {
+  plugins: [
+    postcssHasPseudo(/* pluginOptions */)
+  ]
+}
+```
```

Comment:
Can we add here along these lines, how to integrate with postcss-preset-env? It's trivial to us and we know what's going on but I'm not sure it's clear enough outside of this.

Reply:
Perhaps better on the README

Reply:
I'll add this both in the readme in the `/experimental/` folder and in the readme of this plugin!

Reply:
I added these